### PR TITLE
Have ensemble evaluator methods early return instead of if-else

### PR DESCRIPTION
**Pull request by jonathan-eq**
**Approach**
✂️ 

(Screenshot of new behavior in GUI if applicable)

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: src/ert/ensemble_evaluator/evaluator.py [hunk 1]
```diff
@@ -132,50 +130,54 @@ def _started_handler(self, events: List[CloudEvent]) -> None:
...
-            send_future = asyncio.run_coroutine_threadsafe(
-                self._send_snapshot_update(snapshot_update_event), self._loop
-            )
-            send_future.result()
-            self._signal_cancel()  # let ensemble know it should stop
+            return
+        # if list is empty this call is not triggered by an
+        # event, but as a consequence of some bad state
+        # create a fake event because that's currently the only
+        # api for setting state in the ensemble
```
Comment (jonathan-eq) [inline src/ert/ensemble_evaluator/evaluator.py:171]: I want to change this so that the comment is not necessary.

Comment (berland) [reply src/ert/ensemble_evaluator/evaluator.py:171]: in this PR/commit?

**Comment by berland** [comment]
The commit message can be written more concretely, you are returning early instead of doing if-else.

File: src/ert/ensemble_evaluator/evaluator.py (hunk 1 above)
Comment (jonathan-eq) [reply src/ert/ensemble_evaluator/evaluator.py:171]: I wanted to do it in this PR, but I will do it in a different one

**Review by berland — APPROVED** [review]
👍🏻

**Pull request merged**